examples/tutorials/doc_examples/examples/post_init_notification.py

```diff
@@ -0,0 +1,28 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+# post_init_notification.py --- Example of static notification
```

Comment:
Just spotted this. I changed `"dynamic notification"` to `"static notification"`... to be consistent with the doc.